### PR TITLE
[ADMIN] secrets panel is now tgui

### PR DESCRIPTION
**Pull request by ynot01**
Buttons are color-coded to how they impact the round
Round-impacting buttons without confirmations now have TGUI confirmation as shown in gif below

if you do not have sufficient permissions, the buttons will be greyed out

### CODE REVIEW

File: code/modules/admin/secrets.dm [hunk 1]
```diff
@@ -1,115 +1,66 @@
...
+		var/client/user_client = user
+		holder = user_client //if its a client, assign it to holder
+	else
+		var/mob/user_mob = user
+		holder = user_mob.client //if its a mob, assign the mob's client to holder
+	ui_interact(holder.mob)
+
+/datum/tgui_secrets_panel/ui_data(mob/user)
+	var/list/data = list()
+	var/client/rights = holder
```
Comment (adamsong) [inline code/modules/admin/secrets.dm:35]: Do not give bibby full access to the panel

**Review by adamsong — CHANGES_REQUESTED** [review]

File: code/modules/admin/secrets.dm (hunk 1 above)
Comment (ynot01) [reply code/modules/admin/secrets.dm:35]: fixed

**Comment by JamieD1** [comment]
Can we make it not syndicate? Like I get it haha, but can we just use a centcom background instead?

**Comment by ynot01** [comment]
> Can we make it not syndicate? Like I get it haha, but can we just use a centcom background instead?

done

**Comment by JamieD1** [comment]
> > Can we make it not syndicate? Like I get it haha, but can we just use a centcom background instead?
> 
> done

Got a picture?

**Comment by JamieD1** [comment]
> > Can we make it not syndicate? Like I get it haha, but can we just use a centcom background instead?
> 
> done

Got a picture?

**Comment by ynot01** [comment]
> > > Can we make it not syndicate? Like I get it haha, but can we just use a centcom background instead?
> > 
> > 
> > done
> 
> Got a picture?

the picture has been in the description
image

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**